src/main/java/org/spdx/sbom/gradle/maven/PomInfo.java

```diff
@@ -27,10 +30,21 @@ public interface PomInfo {
 
   URI getHomepage();
 
+  Optional<Organization> getOrganization();
+
+  List<DeveloperInfo> getDevelopers();
+
   @Immutable
   interface LicenseInfo {
     String getUrl();
 
     String getName();
   }
+
+  @Immutable
+  interface DeveloperInfo {
+    Optional<String> getName();
+
+    Optional<String> getEmail();
+  }
```

Comment:
I know this information is out in the wild, but I'm a little bit concerned to include it. @hepwori do we have a position on re-distributing information like this?